### PR TITLE
Add 'npm' subcategory

### PR DESCRIPTION
**Pull request by jneidel**
There are a few npm related apps, which could be grouped together.

This would be the first category 2 levels deep (i.e. Development>Web Development>npm).

### CODE REVIEW

**Comment by jneidel** [comment]
#295 should be included as well

**Comment by sjclayton** [comment]
@jneidel

> This would be the first category 2 levels deep (i.e. Development>Web Development>npm).

This goes directly against: 

> Should only have one level of nested lists, preferably none.

referenced [here](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) under **Requirements for your Awesome List**

**Comment by jneidel** [comment]
> This goes directly against:
> > Should only have one level of nested lists, preferably none.

I already thought this might be a problem...
Do it directly under **Web Development** without the nesting?

**Comment by sjclayton** [comment]
If we do it at all, it should be as another subcategory of **Development**... below **Web Development**..

Ultimately there is nothing stopping us from going 2 levels deep... but if we do that what's to stop us from going even further to the point of it getting way out of hand? :laughing:

**Comment by sjclayton** [comment]
Question --

If this new subcategory is going to be worth our time in implementing, shouldn't all tools related to npm be placed into it?? There are a bunch more on the list... :smile:

**Comment by jneidel** [comment]
Which would you add to it?

There is a bit of overlap with **Release**, but I would keep those there, maybe move the two categories closer to one another.

**Comment by sjclayton** [comment]
@jneidel Looks like you got everything now.... I would maybe move the **npm** subcategory underneath **Release** then it'll be good to go. :100:

**Comment by sjclayton** [comment]
Also consider capitalizing the subcategory name as **NPM**

I know it's not typically capitalized, but I think it'll look weird with the rest of the category headers if we don't.

File: readme.md [hunk 1]
```diff
@@ -155,15 +154,16 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [simplehttp](https://github.com/snwfdhmp/simplehttp) - Most simplest and direct way to start serving a local directory over HTTP from CLI.
 - [shell2http](https://github.com/msoap/shell2http) - HTTP-server to execute shell commands. Designed for development, prototyping or remote control.
 - [HTTP Prompt](https://github.com/eliangcs/http-prompt) - An interactive command-line HTTP client featuring autocomplete and syntax highlighting.
-- [npm-check-updates](https://github.com/tjunnone/npm-check-updates) - Find newer versions of package dependencies than what your package.json allows.
```
Comment (sjclayton) [inline readme.md]: Don't forget to kill this empty line! :stuck_out_tongue:

**Comment by jneidel** [comment]
@sjclayton ready to merge!

**Pull request merged**